src/Compiler/Symbols/Symbols.fsi

```diff
@@ -933,8 +939,8 @@ type FSharpMemberOrFunctionOrValue =
     /// Indicated if this is a value compiled to a method
     member IsValCompiledAsMethod: bool
 
-    /// Indicates if this is a function
-    member IsFunction: bool
```

Comment:
Why was this removed?

Reply:
Strange that tests still passed with this removed....

Reply:
For some reason there were two duplicates, and I removed one of them.

Reply:
Hmmm indicates a bug that some duplicates in signatures are not being detected. Could you add an issue for that please? Thanks

Reply:
@dsyme Done.